Source/Core/Core/HW/GCPadEmu.h

```diff
@@ -8,7 +8,15 @@
 
 #include "InputCommon/ControllerEmu/ControllerEmu.h"
 
+struct GCPadStatus;
```

Comment:
It's allowable to use forward declarations on by-value return types and parameters ([see here](http://stackoverflow.com/a/553869) for a list of valid cases)